README.md

```diff
@@ -96,6 +96,19 @@ realm.write([&car](){
 });
 ```
 
+## Construct a Simple Query
+
+Realm offers an expressive and intuitive way for querying data. Here's a simple example of querying for all `Person` objects and filtering the result based on age.
+
+```cpp
+int main() {
+  auto realm = realm::open<Person, Dog>({.path=path});
+  auto results = realm.objects<Person>().where("age > $0", {20});
+
+  return results.size();
```

Comment:
Hey Marcus - saw you put this up - it's nice to see an example of query here. For what it's worth, I and a couple other folks on the docs team have started working on building out docs for the C++ Alpha and have discovered there's some nuance to using the query results. It would be great to see an example here of doing something with the results, instead of just returning the size.

Reply:
@dacharyc done